src/chart/gauge/GaugeView.ts

```diff
@@ -95,25 +99,26 @@ class GaugeView extends ChartView {
         posInfo: PosInfo
     ) {
         const group = this.group;
-
-        const axisLineModel = seriesModel.getModel('axisLine');
-        const lineStyleModel = axisLineModel.getModel('lineStyle');
-
         const clockwise = seriesModel.get('clockwise');
         let startAngle = -seriesModel.get('startAngle') / 180 * Math.PI;
         let endAngle = -seriesModel.get('endAngle') / 180 * Math.PI;
+        const axisLineModel = seriesModel.getModel('axisLine');
+
+        const roundCap = axisLineModel.get('roundCap');
+        const MainPath = roundCap ? Sausage : graphic.Sector;
```

Comment:
Currently, the `Sector` shape has provided supports to draw corner radius. Using `Sector` should be enough. 
Setting `shape.cornerRadius` to the half of `shape.r` and `shape.innerCornerRadius` to the half of `shape.r0` is identical to `roundCap`(Sausage).

I'm not sure if we should use `Sector` instead of `Sausage`.

Reply:
There is a little difference between setting shape.cornerRadius/shape.innerCornerRadius and `roundCap`(Sausage). 
![image](https://user-images.githubusercontent.com/22974511/95708241-52dc7100-0c8e-11eb-9e39-1a32a6ae4d44.png)

Reply:
Looks like the arc angle is not 360 degrees(a circle)? Is there some wrong with the startAngle and endAngle?
If it's a circle, the path should be closed.

Reply:
Yes，you are right! I fixed the bug that gauge.axisLine can not be a circle when the arc angle is 360 degrees just now.

```
        // angleRangeSpan can't be PI2
        // const angleRangeSpan = (endAngle - startAngle) % PI2
        const angleRangeSpan = !((endAngle - startAngle) % PI2) && endAngle !== startAngle
            ? PI2 : (endAngle - startAngle) % PI2;
```

Now, the path can be closed.

Reply:
I made a test for this and it indicated that no such case. See the screenshot. If I'm missing something, please let me know.

![image](https://user-images.githubusercontent.com/26999792/95718091-8c1ddc80-0ca0-11eb-8ae2-fee7c6305eae.png)

Reply:
The difference between setting shape.cornerRadius/shape.innerCornerRadius and roundCap(Sausage):
1. `axisLine:{color: [color: [[0.9, '#0f0'], [1, '#f00']]}` (cornerRadius maybe better)
![image](https://user-images.githubusercontent.com/22974511/95809887-60533300-0d42-11eb-964a-a6c7b7ea2d6a.png)

2. `porgress:{itemStyle: {color: '#f00',borderColor: '#0f0',borderWidth: 1}}`, and the current value is greater than max value. (roundCap maybe better)
![image](https://user-images.githubusercontent.com/22974511/95810072-b7f19e80-0d42-11eb-9d47-e27efe8e561f.png)

In order to keep `axisLine` and `progress` the same style,  is `roundCap`  better?

Reply:
IMO, `110%` seems not to be a normal value in gauge. Why and when will the percentage exceed 100%?

If we have to display a percentage that exceeds 100%, as you said, maybe it's better to use sausage instead of sector.